### PR TITLE
FTBFS fix for loongarch batch #8

### PR DESCRIPTION
**Pull request by jiegec**
Topic Description
-----------------

- cataclysm: update to 0.G
- carbons: migrate to ab4
- calls: update to 46.0
- feedbackd: new, 0.2.1
    Revived as a dependency of desktop-gnome/calls.
- bspwm: update to 0.9.10 and migrate to ab4

Package(s) Affected
-------------------

- cataclysm: 0.G
- carbons: 0.2.3-1
- bspwm: 0.9.10
- feedbackd: 1:0.2.1
- calls: 46.0

Security Update?
----------------

No

Build Order
-----------

```
#buildit bspwm feedbackd calls carbons cataclysm
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

**Experimental Architectures**

- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`

### CODE REVIEW

File: app-web/carbons/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,5 @@
+abinfo "Building carbons ..."
+make FLAGS="$CFLAGS"
+
+abinfo "Installing carbons ..."
+make install DESTDIR="$PKGDIR"
```
Comment (MingcongBai) [inline app-web/carbons/autobuild/build:5]: Is `PURPLE_PLUGIN_DIR` specified?

**Review by MingcongBai — APPROVED** [review]

**Review by MingcongBai — CHANGES_REQUESTED** [review]
See above.

File: app-web/carbons/autobuild/build (hunk 1 above)
Comment (jiegec) [reply app-web/carbons/autobuild/build:5]: The path is no longer required to pass. The default value is good. See https://github.com/gkdr/carbons/blob/2fa542db43631bfef8638d686b3361539ee818e2/Makefile#L17 and https://github.com/gkdr/carbons/blob/2fa542db43631bfef8638d686b3361539ee818e2/Makefile#L114

Comment (jiegec) [reply app-web/carbons/autobuild/build:5]: Actually, the previous path was wrong after upstream added $(DESTDIR):

carbons upgraded from 0.2.3 to 0.2.3-1 on arm64:
<details>
<summary>3 added, 11 removed</summary>

```diff
--- a
+++ b
@@ -1,17 +1,9 @@
 drwxr-xr-x     ./
 drwxr-xr-x     ./usr/
+drwxr-xr-x     ./usr/lib/
+drwxr-xr-x     ./usr/lib/purple-2/
+-rw-r--r--     ./usr/lib/purple-2/carbons.so
 drwxr-xr-x     ./usr/share/
 drwxr-xr-x     ./usr/share/doc/
 drwxr-xr-x     ./usr/share/doc/carbons/
 -rw-r--r--     ./usr/share/doc/carbons/LICENSE
-drwxr-xr-x     ./var/
-drwxr-xr-x     ./var/cache/
-drwxr-xr-x     ./var/cache/acbs/
-drwxr-xr-x     ./var/cache/acbs/build/
-drwxr-xr-x     ./var/cache/acbs/build/acbs.qfhlh6s3/
-drwxr-xr-x     ./var/cache/acbs/build/acbs.qfhlh6s3/carbons-0.2.3/
-drwxr-xr-x     ./var/cache/acbs/build/acbs.qfhlh6s3/carbons-0.2.3/abdist/
-drwxr-xr-x     ./var/cache/acbs/build/acbs.qfhlh6s3/carbons-0.2.3/abdist/usr/
-drwxr-xr-x     ./var/cache/acbs/build/acbs.qfhlh6s3/carbons-0.2.3/abdist/usr/lib/
-drwxr-xr-x     ./var/cache/acbs/build/acbs.qfhlh6s3/carbons-0.2.3/abdist/usr/lib/purple-2/
--rw-r--r--     ./var/cache/acbs/build/acbs.qfhlh6s3/carbons-0.2.3/abdist/usr/lib/purple-2/carbons.so

```

**Comment by jiegec** [comment]
ppc64el build depends on #5533.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**